uniq/src/cli.rs

```diff
@@ -0,0 +1,53 @@
+use clap::{
+    crate_authors, crate_description, crate_name, crate_version, App, AppSettings::ColoredHelp, Arg,
+};
+
+pub(crate) fn create_app<'a, 'b>() -> App<'a, 'b> {
+    App::new(crate_name!())
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .help_message("Display help information.")
+        .version_message("Display version information.")
+        .help_short("?")
+        .settings(&[ColoredHelp])
+        .arg(
+            Arg::with_name("INPUT")
+                .help("Input file (defaults to stdin).")
+                .index(1)
+                .default_value("-"),
+        )
+        .arg(
+            Arg::with_name("OUTPUT")
+                .help("Output file (defaults to stdout).")
+                .index(2)
+                .default_value("-"),
+        )
+        .arg(
+            Arg::with_name("count")
+                .help("Prefix lines by the number of occurrences")
+                .short("c")
+                .long("count"),
+        )
+        .arg(
+            Arg::with_name("repeated")
+                .help("Only print duplicate lines, one for each group")
+                .short("d")
+                .long("repeated"),
+        )
+        .arg(
+            Arg::with_name("skip-fields")
+                .help("Avoid comparing the first N fields")
+                .short("f")
+                .long("skip-fields")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("skip-chars")
+                .help("Avoid comparing the first N characters")
+                .short("s")
+                .long("skip-chars")
+                .takes_value(true),
```

Comment:
Use `value_name` function instead, it implies `takes_value` and add a name to show in the help message.
Otherwise the help message will be `-s, --skip-chars <skip-chars>      Avoid comparing the first N characters`